Update ssh_utils.py

Viocalpan opened the pull request:
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community
* [] Fix quit bug

pivotal-issuemaster commented:
@Viocalpan Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=11078)!

[Click here](https://cla.pivotal.io/sync/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=11078) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

Viocalpan commented:
fix quit bug

Viocalpan commented:
def escapeLine(self, line):
&nbsp;&nbsp;&nbsp; '''Escape occurrences of \ and $ as needed and package the line as an "eval" shell command'''
&nbsp;&nbsp;&nbsp; line = line.strip()
&nbsp;&nbsp;&nbsp; if line == 'EOF' or line == 'exit' or line == 'quit':
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; self.close()&nbsp; #Add by Viocal pan 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; raise self.SessionCmdExit()
&nbsp;&nbsp;&nbsp; line = line.split('\\')
&nbsp;&nbsp;&nbsp; line = '\\\\'.join(line)
&nbsp;&nbsp;&nbsp; line = line.split('"')
&nbsp;&nbsp;&nbsp; line = '\\"'.join(line)
&nbsp;&nbsp;&nbsp; line = line.split('$')
&nbsp;&nbsp;&nbsp; line = '\\$'.join(line)
&nbsp;&nbsp;&nbsp; line = 'eval "' + line + '" < /dev/null'
&nbsp;&nbsp;&nbsp; return line
